src/network.h

```diff
@@ -17,6 +17,8 @@ enum {
   EPOLL_PEER_FD = 2,   // mark epoll events for peer connections with this code
   EPOLL_TIMEOUT =
       1000 // mark epoll to block this many ms until event is recieved
+  MAX_SIZE_MESSAGE_INT = ((1024 * 1024) / 32) + 2 // 1 Mib + 2 ints for overhead
+  //this is a little more than needed because the type is only one byte
 };
```

Comment:
you need a comma there it's likely why it's not compiling correctly 
```suggestion
  EPOLL_TIMEOUT =
      1000, // mark epoll to block this many ms until event is recieved
  MAX_SIZE_MESSAGE_INT = ((1024 * 1024) / 32) + 2 // 1 Mib + 2 ints for overhead
  //this is a little more than needed because the type is only one byte
};
```